src/users/entities/user.entity.ts

```diff
@@ -44,12 +45,11 @@ export class User {
   @DeleteDateColumn()
   deleted_at: Date;
 
-  @OneToMany(() => Post, post => post.user)
-  posts?: Post[];
+  @OneToMany(() => PostEntity, post => post.user)
+  posts?: PostEntity[];
```

Comment:
gắn thêm option `nullable: true` vào decorator là được em
```suggestion
@OneToMany(() => PostEntity, post => post.user, { nullable: true })
  posts: PostEntity[];
```